package.json

```diff
@@ -29,7 +29,7 @@
   },
   "dependencies": {
     "@fortawesome/fontawesome-free-webfonts": "^1.0.9",
-    "@spinnaker/kayenta": "0.0.62",
+    "@spinnaker/kayenta": "^0.0.63",
```

Comment:
Just confirming that we want to add the caret rather than pin an exact version -- I think `^0.0.63` will update to most recent `0.x.y`

Reply:
doh, good spot.  I did not intend a range.  Will update.